internal/datastore/postgres/common/bulk.go

```diff
@@ -44,6 +45,12 @@ func (tg *tupleSourceAdapter) Values() ([]any, error) {
 	tg.valuesBuffer[6] = caveatName
 	tg.valuesBuffer[7] = caveatContext
 
+	if len(tg.colNames) > 8 && tg.current.Integrity != nil {
```

Comment:
I thought we weren't going to add this for anything except crdb. Am I mistaken?

Reply:
We aren't, but this is shared with CRDB, so I needed to handle it here